src/main/scala/cromwell/binding/WdlBinding.scala

```diff
@@ -115,6 +114,11 @@ object WdlBinding {
     }
   }
 
+  /* All MemberAccess ASTs that are not contained in other MemberAccess ASTs */
+  def findTopLevelMemberAccesses(expr: AstNode) = WdlBinding.findAstsWithTrail(expr, "MemberAccess").filter {
+    case (k, v) => !(v map {case a:Ast => a.getName} contains "MemberAccess")
```

Comment:
I'd like to figure out what Cromwell should be doing stylistically here.  @geoffjentry could you possibly scare up references to any style guides you feel would apply?  Personally I feel like I'm starting to develop habits writing code like this, and if they're not the right habits I'd like to correct that ASAP. :smile: